packages/web-react/src/components/Modal/README.md

```diff
@@ -189,6 +192,10 @@ secondary actions are optional.
 👉 Please note the actions are _visually_ ordered from right to left from the tablet breakpoint up. However, the
 _actual_ order in code is followed when users tab over the interface.
 
+ℹ️ We strongly recommend always providing the footer and at least a primary action. This way, users can
```

Comment:
`We strongly recommend always providing` - this sounds weird to me. May I suggest another copy? `We strongly recommend that the footer should be provided in every use case and at least with the primary action button.` What do you think?